.travis.yml

```diff
@@ -1,43 +1,70 @@
 language: php
-
 sudo: false
+dist: trusty
 
-cache:
-  directories:
-    - $HOME/.composer/cache/files
+services:
+  - docker
+
+addons:
+  chrome: stable
 
-php: [5.4, 5.5, 5.6, 7.0, 7.1, 7.2]
+php:
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
 
 env:
   global:
-    - WEBDRIVER=selenium
+    - DISPLAY=:99.0
+    - BROWSER_NAME="chrome"
+    - DRIVER_OPTIONS='{"args":["headless", "no-sandbox","window-size=1024,768"]}'
+    - CHROMEDRIVER_VERSION="2.38"
+    - START_XVFB="0"
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+    - jar
 
 matrix:
-  fast_finish: true
   include:
-    - php: 7.0
-      env: WEBDRIVER=selenium-remote
```

Comment:
can we keep a test corresponding to that case ? It was important to test for people relying on Saucelabs to run their Selenium